ui/src/app/utils/research-purpose.tsx

```diff
@@ -58,6 +62,15 @@ export function getSelectedPopulations(researchPurpose: ResearchPurpose) {
     .subCategory.filter(item => researchPurpose.populationDetails.includes(item.shortName)).length > 0));
   categories.filter(category => category.subCategory = category.subCategory
     .filter(subCategoryItem => researchPurpose.populationDetails.includes(subCategoryItem.shortName)));
+  if(researchPurpose.otherPopulationDetails !== '') {
```

Comment:
Can you add a comment here? Why does the above filtering/mapping from SpecificPopuplationItems not capture this case?

Reply:
^